### DISCUSSION TITLE
Key popup background color

### DISCUSSION THREAD

**Discussion by Ansarmughal99** [itsmartashub/floristyle#18] [Ideas]
While the key popup keys color is always white and is not noticeable if keys are popping or not .
Key popup bg color has to be the same just like the keyboard dynamic color.
Also the keyboard bg color is dull as compared to other keyboards using dynamic color.

**Comment by itsmartashub** [comment]
Please checkout [v5.0.0](https://github.com/itsmartashub/floristyle/releases/tag/v5.0.0) version
